extraction/text_extractor.py

```diff
@@ -0,0 +1,33 @@
+"""This script checks whether DOM has text tag or not and
+creates and returns the Text object"""
+
+import bs4
+
+from extraction.data_models.text import Text
+from extraction.interface_content_extractor import IContentExtractor
+from extraction.utils import string_utils as utils
+
+TEXT_TAGS = ['p', 'span', 'code'] + utils.HEADING_TAGS
+TEXT_MIN_SCORE = utils.TEXT_MIN_SCORE
```

Comment:
TEXT_TAGS could be moved to utils as well.

Reply:
Thanks for suggestion! Will move TEXT_TAGS and utils and use both TEXT_MIN_SCORE and TEXT_TAGS directly into function instead of re-initializing in text_extractor.